src/System.Net.Http/ref/System.Net.Http.cs

```diff
@@ -386,8 +386,10 @@ public sealed partial class HttpContentHeaders : System.Net.Http.Headers.HttpHea
         public byte[] ContentMD5 { get { throw null; } set { } }
         public System.Net.Http.Headers.ContentRangeHeaderValue ContentRange { get { throw null; } set { } }
         public System.Net.Http.Headers.MediaTypeHeaderValue ContentType { get { throw null; } set { } }
+        protected override void EnsureHeaders() { }
         public System.Nullable<System.DateTimeOffset> Expires { get { throw null; } set { } }
         public System.Nullable<System.DateTimeOffset> LastModified { get { throw null; } set { } }
+        public override string ToString() { throw null; }
```

Comment:
Technically you don't need these overrides in the ref.  The only place they'd impact codegen is for a type deriving from this and calling to its base, but the type is sealed.